torch/_inductor/select_algorithm.py

```diff
@@ -1115,7 +1117,16 @@ def get_timings():
         if timings == {} or choices[0] not in timings:
             return choices[0].output_node()
 
-        selected_choice = builtins.min(timings, key=timings.__getitem__).output_node()
+        selected_key = builtins.min(timings, key=timings.__getitem__)
+        selected_time = timings[selected_key]
+        if (
+            (not isinstance(selected_time, float))
+            or (selected_time < 0.0)
+            or (not math.isfinite(selected_time))
+            or math.isnan(selected_time)
```

Comment:
feels a little overkill (pretty sure we only generate inf and regular floats) but not a big deal